srtcore/queue.h

```diff
@@ -385,6 +385,11 @@ friend class CUDTUnited;
    int ioctlQuery(int type) const { return m_pChannel->ioctlQuery(type); }
    int sockoptQuery(int level, int type) const { return m_pChannel->sockoptQuery(level, type); }
 
+   void silence()
+   {
+       m_bClosing = true;
+   }
+
```

Comment:
Not sure if this is a good name for this function. I would recommend something like
```
void close();
```
or
```
void setClosing(bool val = true);
```

Reply:
Could be. I think `setClosing` may sound better, although this parameter is not necessary. The `true` value is set when created and all the time until it's set to false, and this state is expected to persist until destruction. So far it was only set in destructor.